packages/icon/src/types/LGGlyph.ts

```diff
@@ -3,6 +3,7 @@ import * as SVGR from './SVGR';
 
 export interface ComponentProps extends SVGR.ComponentProps {
   size?: number | 'small' | 'default' | 'large' | 'xlarge';
+  ['data-testid']?: string;
```

Comment:
Why do we need to include this?

Reply:
Was causing an issue here: https://github.com/mongodb/leafygreen-ui/pull/1512/files#diff-999a04b89f74f252924b0289efa25728ca194ac073530555f9abdbbfd3e904b3L123

Reply:
the link seems to be broken, its doesn't show the line or file,

Reply:
SideNavGroup line 123!